lib/app_manifest.rb

```diff
@@ -36,9 +36,9 @@ def canonicalize_env(manifest)
       canonicalize_key(manifest, :env) do |env|
         Hash[
           env.map do |key, value|
-            if value.is_a? String
+            if !value.is_a?(Hash)
               value = {
-                value: value,
+                value: value.to_s,
```

Comment:
I think we should be careful about coercing types of values that we do not accept in Nyata.